src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router'
+import React from 'react'
+
+const Breadcrumbs = props => {
```

Comment:
i do mildly prefer, just because i think it helps set the stage, but it's fine for now